ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -2982,6 +2982,17 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-nutanix-ipi-proxy
+- as: nutanix-ipi-registry-nutanix-object-p2-f28
+  cluster: build01
+  cron: 30 10 15 * *
```

Comment:
Strange I got following cron setting:
$ ./generate-cron-entry.sh nutanix-ipi-registry-nutanix-object-p2-f28 openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml
cron: 15 4 10 * *

Reply:
@sgaoshang Hi, sorry for the delay response. so I should update as your result, right?

Reply:
I think so but pls double confirm it.